contracts/src/v0.8/ccip/DonIdClaimer.sol

```diff
@@ -0,0 +1,39 @@
+// SPDX-License-Identifier: BUSL-1.1
+pragma solidity ^0.8.24;
+
+import {ICapabilityConfiguration} from "../keystone/interfaces/ICapabilityConfiguration.sol";
+
+contract DonIDClaimer {
+  error ZeroAddressNotAllowed();
+
+  uint256 public donID;  
+  address internal immutable i_capabilitiesRegistry;
+
+  constructor(
+    address capabilitiesRegistry
+  ) {
+    if (capabilitiesRegistry == address(0)) {
+      revert ZeroAddressNotAllowed();
+    }
+    i_capabilitiesRegistry = capabilitiesRegistry;
```

Comment:
Let's fetch the next don ID upon construction:

```suggestion
    i_capabilitiesRegistry = capabilitiesRegistry;
    s_nextDONId = i_capabilitiesRegistry.getNextDONId()
```

This also ensures that the address is correct upon deployment. If the address doesn't support that call, the deployment would revert.

Reply:
https://github.com/smartcontractkit/chainlink/pull/16926/commits/e5e4a9ed063b98f455d7792e02a2adf5da36ca33